src/gnrc/netapi.rs

```diff
@@ -21,3 +24,11 @@ pub fn dispatch_send(
     }
     subscribers
 }
+
+/// Joins an IPV6 multicast group at the provided address
+pub fn join_multicast_v6(pid: KernelPID, addr: &[u8; 16]) {
```

Comment:
embedded_nal_async also has the Ipv6Addr type. Wouldn't it make more sense to use this one? Using core::net::Ipv6Addr, we would have two different types then in riot-wrappers for the same purpose.
If I use embedded-nal in the application code, I have to convert it to core::net::Ipv6Addr which is a bit unhandy.
Suggestion: If `with_embedded_nal` feature is activated, `pub use embedded_nal::Ipv6Addr`, otherwise use the one from core::net?